src/beman/optional26/tests/optional_range_support.t.cpp

```diff
@@ -29,6 +29,10 @@
 #include <unordered_set>
 #include <vector>
 
+#if !defined(__clang__) && defined(__GNUC__) && (__GNUC__ < 12)
```

Comment:
I don't like this solution. I would try to see exactly what gcc minor version we have on CI (you can add logs in yaml file).

Maybe it's worth considering upgrade to a newer gcc11 minor version or to remove gcc11 support. 

@steve-downey , do you have any input here?

Reply:
I found that this is probably [GCC Bugzilla PR89074](https://gcc.gnu.org/PR89074). The bugfix landed in GCC 12.0 and wasn't backported to GCC 11 releases. So perhaps it's OK to only check the major version? (But perhaps the macro name should be more precise.)

Reply:
I would be ok dropping 11 if we can't find a straightforward work around.

Reply:
My $.02 -- drop gcc 11 is circa  mid-2021. We're living on the cutting edge here.  I'd have a different opinion if all the later compilers didn't just handle this.

Reply:
I'm now switching to remove GCC 11 coverage from CI, which should enable removing all such skippings.

Reply:
Cool. I'll make sure to remember this decision for exemplar.